LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -353,8 +353,13 @@ - (void)_setupFingerPrint {
     if (!self.biometricsContext && _allowUnlockWithBiometrics && !_useFallbackPasscode) {
         self.biometricsContext = [[LAContext alloc] init];
         
+        LAPolicy policy = LAPolicyDeviceOwnerAuthenticationWithBiometrics;
+        if (@available(iOS 9.0, *)) {
```

Comment:
Why is this like this? I don’t have much experience with `LAPolicy`.

Reply:
LAPolicyDeviceOwnerAuthentication was added in iOS 9, it gives the native biometrics dialogue a fallback. If you fail the first try, it shows the ability to use your system passcode instead.

I'll update my pull request.